### DISCUSSION TITLE
How to proxy in a nodejs/express app or a static website

### DISCUSSION THREAD

**Discussion by a4z** [plausible/analytics#2236] [Anything Else]
Hi, I am trying to understand the proxy documentation
https://plausible.io/docs/proxy/introduction

but being not a web dev I have some problems. 

I have 2 apps, one is an nodejs / express app, and the other site is static.

I think I understand how to do that part
`https://<yourdomain.com>/js/script.js -> https://plausible.io/js/script.js`
(Just download script.js into my js folder and load it from there)

But I do not know how to do that
`https://<yourdomain.com>/api/event    -> https://plausible.io/api/event`
I would need to have a route and forward the call from my api/event endpoint to plausible , with all parameters attached

How could that work is a nodejs/express app, and for a static website?

**Comment by metmarkosaric** [comment]
hi @a4z! the proxy that's very easy to start with for anyone regardless of the technology used to build the site is the Cloudflare proxy. that's why we recommend it and it takes a couple of minutes to setup with [our simple step by step instructions](https://plausible.io/docs/proxy/guides/cloudflare).

**Reply by a4z** [reply]
I was able to do that, and this seems to work!
Great tutorial, makes me feel like a pro 😀 
Thanks a lot!
